src/PolyhedralGeometry/helpers.jl

```diff
@@ -188,6 +192,10 @@ function stack(A::Vector{Polymake.Vector{Polymake.Rational}})
 end
 =#
 
+ambient_dim(x::AbstractVector) = length(x)
+ambient_dim(x::AbstractMatrix) = size(x, 2)
```

Comment:
```suggestion
ambient_dim(x::AbstractMatrix) = ncols(x)
```
I understand the rationale but I somehow feel uneasy about associating mathematical properties like this to raw Julia matrices and vectors. But perhaps that's just me needing a strong stomach :-). Anyway: CC @fieker @thofma

Reply:
Yes, I agree with @fingolfin.

Reply:
At the moment these are intended for internal usage only, in context of their specific interpretations. Benjamin also agrees, so I will add an underscore.